formats/placeholder/frontend/examples/organisation.json

```diff
@@ -0,0 +1,36 @@
+{
+  "base_path": "/government/organisations/bona-vacantia",
+  "content_id": "c9109663-0128-4db4-b513-a9cdfb691034",
+  "title": "Bona Vacantia",
+  "format": "placeholder_organisation",
+  "document_type": "organisation",
+  "schema_name": "placeholder",
```

Comment:
When you send something to publishing-api you can specify `format` OR `document_type` & `schema_name`.

In the last case, `format` will be set to `schema_name`. So the example could read:

```
+  "format": "placeholder_organisation",
+  "document_type": "organisation",
+  "schema_name": "placeholder_organisation",
```

or

```
+  "format": "placeholder",
+  "document_type": "organisation",
+  "schema_name": "placeholder",
```